mslib/mswms/mswms.py

```diff
@@ -46,6 +46,14 @@ def main():
     parser.add_argument("--logfile", help="If set to a name log output goes to that file", dest="logfile",
                         default=None)
     parser.add_argument("--update", help="Updates MSS to the newest version", action="store_true", default=False)
+
+    subparsers = parser.add_subparsers(help='Available actions', dest='action')
+    gallery = subparsers.add_parser("gallery", help="Generates plots of all layers not already present")
+    gallery.add_argument("--force-regenerate", action="store_true", default=False,
```

Comment:
this naming let assume that there could be also a  --init or --seed to start with.

a shorter word for force-regenerate is refresh. And a common naming approach is 
- create
- clean
- refresh would do first a clean and a create afterwards

We do need clean too, to get to an origin state back.

Reply:
Someone who wants to serve these static files by a nginx or apache2 wants to have an option to give a dir name where the /static/plots are stored

for explaination
https://code.google.com/archive/p/modwsgi/wikis/ConfigurationGuidelines.wiki#Hosting_Of_Static_Files

Reply:
Parameters were renamed.
The gallery is now stored inside the _gallerypath defined in mss_wms_settings, which defaults to the mss_wms_settings location.